static-assets/components/cstudio-common/common-api.js

```diff
@@ -1776,7 +1776,7 @@ var nodeOpen = false,
                     openDialogCb);
             },
 
-            performSimpleIceEdit: function (item, field, isEdit, callback, aux, isFlattenedInclude) {
+            performSimpleIceEdit: function (item, field, isEdit, callback, aux, isFlattenedInclude, display) {
```

Comment:
Rename to _openHidden_ encapsulate the logic of the visibility property. Give the param a default fallback value to false for backward compatibility.